frontend/src/app/testQueue/TestCardForm/TestCardForm.utils.tsx

```diff
@@ -183,6 +182,24 @@ export const hasAnySupportedDiseaseTests = (
   }
   return false;
 };
+export const whichAoeFormToDisplay = (
```

Comment:
The switch logic is fine for now but I do see the number of cases growing quite fast as we add more diseases. Going forward we should probably think about how to generalize this for n diseases.